pythonFiles/include/blender_vscode/operators/addon_update.py

```diff
@@ -36,7 +37,14 @@ def execute(self, context):
         return {'FINISHED'}
 
 def reload_addon_action(data):
-    for name in data["names"]:
+    module_names = []
+    for name, dir in zip(data["names"], data["dirs"]):
+        if is_addon_legacy(Path(dir)):
+            module_names.append(name)
+        else:
+            module_names.append("bl_ext.user_default." + name)
```

Comment:
Seems fine for now, but I wonder if the `user_default` part should be derived from the path somehow.

Reply:
I wondered about that, but I wasn't sure if there was a situation where you would want the extension to be installed in a folder other than `4.2/extensions/user_default`? And if there is then how to specify it.